operator/gitops/argocd/pipeline-service/kustomization.yaml

```diff
@@ -6,7 +6,7 @@ resources:
   - openshift-pipelines
   - pipelines-as-code
   - tekton-chains
-  - tekton-results
+  - tekton-results/overlays/dev
```

Comment:
@Roming22 I think this may present a problem when we try to get the change applied to infra-deployments. We may need to discuss with the core team or the GitOps team on how we manage overlays for AWS services.

Reply:
good catch @adambkaplan / I would agree

this stems from @AndrienkoAleksandr 's original commit, so we may want to cross verify with him in the end, but at first glance, given the moves he made into the base subdir, feels like this should be `tekton-results/base` and then the credential patches needed for the developer flow should be called directly from the ci-runner flow

infra-deployments definitely has the pattern of accessing base dirs ... see https://github.com/redhat-appstudio/infra-deployments/blob/main/components/jvm-build-service/kustomization.yaml#L3 for example

I'll see if I can do that, first in a separate commit, and we get clean CI,, then go from there

with all that, I'm certainly do not disagree wrt getting input from core/gitops in conjunction

Reply:
btw looking at the time stamps of the comments, it looks like @adambkaplan 's original comment here was edited after my response (54 minutes for his vs. 37 for mine as of this comment).

I think that may be how our duplicate refs to `/base` arose :-)

Reply:
switching to base and removing overlays/dev is passing the ci tests

marking resolved